projects/com.oracle.truffle.llvm.nodes.impl/src/com/oracle/truffle/llvm/nodes/impl/cast/LLVMToI8Node.java

```diff
@@ -132,6 +132,11 @@ public byte executeI8(LLVM80BitFloat from) {
     public abstract static class LLVMAddressToI8Node extends LLVMToI8Node {
```

Comment:
I thought that `LLVMAddressNode` included Truffle objects? And `Boolean` is a bit like that - a managed object.